### ISSUE TITLE
Document (and configure?) issue reporting template

### ISSUE THREAD

**Issue by sscaling** [wurstmeister/kafka-docker#430]
We get multiple issues a week that are generally lacking in information. To help the community provide better information when raising an issue we should configure issue templates and document what the process is (along with any expectations - i.e. will be closed after 30 days if no activity etc)

**Comment by barrer** [comment]
I agree with this view, many questions can be searched for existing issues. I think the following knowledge needs to be reserved before using this image: docker, docker network, docker compose, zookeeper, kafka, network operating system. It is very difficult for novices.
